sdks/python/apache_beam/coders/coders.py

```diff
@@ -386,8 +386,14 @@ def _create_impl(self):
 class BytesCoder(FastCoder):
   """Byte string coder."""
 
+  def __init__(self, encode_utf8=False):
```

Comment:
We shouldn't be adding a parameter to BytesCoder--if we want utf-8 encoding StrUtf8Coder should be used instead.

Reply:
Ah, good point.  That may change things on the Java side though, since previously this code was using the bytes coder.

Reply:
The String utf8 coder was added after the initial cross-language configuration version. For TODO see https://jira.apache.org/jira/browse/BEAM-7244

Reply:
removed changes to `BytesCoder` and replaced usage with `StrUtf8Coder`.